CHANGELOG.next.asciidoc

```diff
@@ -111,6 +111,7 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - Fix issue where old data could be saved in the memory queue after acknowledgment, increasing memory use {pull}41356[41356]
 - Ensure Elasticsearch output can always recover from network errors {pull}40794[40794]
 - Add `translate_ldap_attribute` processor. {pull}41472[41472]
+- Errors resulting from reading on closed connections are suppressed {issue}40824[40824] 
```

Comment:
I think it's better to emphasize that it was rather a fix of the log message flood.

```suggestion
- Remove unnecessary debug logs during idle connection teardown {issue}40824[40824]
```